packages/SidePanel/src/components/Footer/Footer.styles.js

```diff
@@ -1,10 +1,32 @@
-import { css } from "styled-components";
+import styled, { css, keyframes } from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens/lib/tokens";
 
+function slideIn() {
+  return keyframes`
+    0% { width: 100%; }
+    100% { width: 100%; }
+  `;
+}
+
+function slideOut(width) {
+  return keyframes`
+    0% { width: 100%; }
```

Comment:
wait, shouldn't we be animating the position using `transform: translate` instead of animating the width value? Also couldn't' these be helper functions? 
I notice we have this function as well taht could also use something similar it perhaps? 
https://github.com/acl-services/paprika/blob/master/packages/SidePanel/src/components/Dialog/Dialog.styles.js#L18

Reply:
Actually why is the footer animated anyway? Isn't the the container dialog animated in? Why does this element need its own separate animation? Shouldn't it always be `width:100%` of the container Dialog compoennt?

Reply:
The issue with footer was because its `width` was `33%` all the time, but somehow when it's animating, it counts `33%` of the side panel, not `33%` of `body` 
![image](https://user-images.githubusercontent.com/38733362/74190821-d299d380-4c07-11ea-842e-c29bf2af7758.png)

I think that's why it was shifting, not because of `translate`

Reply:
In the dom it's like:
```
<dialog> //position fixed to container
 <header>
 <content>
 <footer>  //position fixed to container if it's a sticky footer
<dialog>
```
We're just animating the `dialog`, but when the `footer` is sticky, we have to animate it at the same time, otherwise it looks like shifting

Reply:
I just wonder if there is a way around this without needing to animate the footer as well as the sidepanel itself. Can the footer be absolute positioned at the bottom of a container div that is `height:100%` instead perhaps? Then it woudn't need to be `position:fixed`. Would need some time to play about with that but i think could work

Reply:
Do you think this is better? 
![image](https://user-images.githubusercontent.com/38733362/74201804-91152280-4c1f-11ea-8b4a-9b83f32c6ed8.png)
But this will be a breaking change since I added 2 wrappers

Reply:
I don't see what that does can you commit the code so i can see how it affects it?